setup.py

```diff
@@ -0,0 +1,11 @@
+from setuptools import setup, find_packages
```

Comment:
Thanks for turning this into a package, I think that really streamlines things. 

Could you add something to the README about how to install it? I think just the standard `python setup.py install` should be fine

Reply:
Also, do you see a difference in the way deployment would happen because of this? I would think that deployment looks more like a `git pull && python setup.py install` rather than just re-pulling